### PR TITLE
[refactor] refactor directory structure

### PR DESCRIPTION
**Pull request by xingchensong**
cc @pengzhendong  @robin1001

### CODE REVIEW

File: itn/chinese/inverse_normalizer.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022 Zhendong Peng (sxc19@tsinghua.org.cn)
```
Comment (pengzhendong) [inline itn/chinese/inverse_normalizer.py:1]: Fix author name.

File: itn/processor.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022 Zhendong Peng (sxc19@tsinghua.org.cn)
```
Comment (pengzhendong) [inline itn/processor.py:1]: See above.

File: itn/token_parser.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022 Zhendong Peng (sxc19@tsinghua.org.cn)
```
Comment (pengzhendong) [inline itn/token_parser.py:1]: See above.

**Review by pengzhendong — CHANGES_REQUESTED** [review]

File: itn/chinese/inverse_normalizer.py (hunk 1 above)
Comment (xingchensong) [reply itn/chinese/inverse_normalizer.py]: Done.

**Review by pengzhendong — APPROVED** [review]

**Pull request merged**